buddyboss/buddyboss-platform
# Fixed - YouTube video is not embedded/showing on the profile timeline, private messaging.

### All Submissions:






### Changes proposed in this Pull Request:

I have added a filter for embedded videos in bp-activity-filters.php and bp-messages-filters.php

Fixes #2049.

### How to test the changes in this Pull Request:

1. Go to the Activity section - front side
2. Enter Youtube URL like - https://www.youtube.com/watch?v=luNts-JE8Ms there.
3. See (Video will embedded/showing)
4. The Same thing is happening in the message section.

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crVDlq9131

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
